docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,296 +97,193 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
 
-. For *Axis side*, click *Right*, then click *Close*.
+When you drag a text or IP address field into the *Lens* workspace you will see
+the most frequent values of that field using the *top values* function. The values
+are ranked based on another metric like *Count of records*. Only exact matches count:
+`win 10` and `win xp` are distinct.
 
-[float]
-[[change-the-visualization-type]]
-==== Change the visualization type
+For this tutorial, you have decided to see the most frequently viewed URLs on your website
+ranked by the `Unique count of clientip` metric, but you have not yet determined which chart type to use.
 
-. In the editor, click *Average of bytes* in the area chart layer. 
+. Starting with the default bar chart, drag the `clientip` field onto the *Vertical axis*.
+This will produce the `Unique count of clientip` metric. Do not drop the field into the main workspace,
+because the *top values* function will be used instead.
 
-. From the *Value format* dropdown, select *Bytes (1024)*, then click *Close*.
+. Drag and drop `request.keyword` to the main workspace, and *Lens* will add `Top value of request.keyword`
+to the *Horizontal axis*.
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_4.png[Multiaxis chart]
+image::images/lens_end_to_end_2_1_1.png[Vertical bar chart with top values of request.keyword by most unique visitors]
 
-[discrete]
-[[lens-legend-position]]
-==== Change the legend position
+This chart is hard to read because the `request.keyword` field contains long text. You could try
+using one of the *Suggestions*, but the suggestions also have issues with long text. Instead, switch
+to a *Table* visualization:
 
-The visualization is done, but the legend uses a lot of space. Change the legend position to the top of the chart.
+. Click *Bar vertical stacked* to open the chart switcher
 
-. From the *Legend* dropdown, select the top position.
+. Click *Table*
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_5.png[legend position]
+image::images/lens_end_to_end_2_1_2.png[Table with top values of request.keyword by most unique visitors]
 
-. Click *Save and return*.
+Customize the table even more:
 
-[discrete]
-[[percentage-stacked-area]]
-=== View the health of your website 
+. Click *Top values of request.keyword*
 
-To detect unusual traffic, bad website links, and server errors, create a percentage stacked area chart that displays the associated response codes.
+.. Increase the *Number of values*. The maximum allowed value is 1000.
 
-. Open *Lens*.
+.. In the *Display name* field, enter `Page URL`.
 
-. From the *Available fields* list, drag and drop the data fields to the *Drop a field or click to add* fields in the editor.
+. Click *Close*.
 
-.. Drag and drop *Records* to the *Vertical axis* field.
+. Click *Save and return*.
 
-.. Drag and drop *@timestamp* to the *Horizontal axis* field.
+. Arrange the table panel so that it has its own row, but keep the default width of half the screen.
```

Comment:
Are users arranging the panel, or moving the panel? 

Why does it matter that the panel width is half the screen?

Also, according to the Elastic style guide, we never use `screen` in the docs. I would replace all references to `screen` with `dashboard`.